app/renderer/abr-document.js

```diff
@@ -350,6 +366,65 @@ AbrDocument.prototype = {
         return this.save(path, callback);
     },
 
+    initWatcher: function () {
+        var that = this;
+        var updateTitle = function () {
```

Comment:
Ok I see.
I didn't really like this anyway, because having two functions like that can be confusing.
I'll remove the duplicate function.